merlion/models/forecast/sklearn_base.py

```diff
@@ -278,4 +298,4 @@ def _get_immedidate_forecasting_prior(self, data):
         if isinstance(data, TimeSeries):
             data = data.to_pd()
         data = data.values
-        return data[-self.maxlags :].reshape(-1, order="F")
```

Comment:
I used `order="F"` to have the flattened array place the data of the same sequence altogether, so it is like [first sequence, second sequence, ...] this is making my debug easier (I can locate those sequences in the rolling window much easier than if it it flattened along the time axis). 
Since we have confirmed that the manipulation is correct, as long as RollingWindow class and sklearn_base used the same reshape order, it should be fine. 

Do we run unittest to confirm the values are the same?

Reply:
I've confirmed that the values are the same with equivalent parameter settings. Actually I was running into bugs when I called `x.reshape(..., order="F")` multiple times on a single array (I think each subsequent call changes the underlying data order), so this is the reason I changed it. This method was easier for me to understand the logic of.